setup.cfg

```diff
@@ -0,0 +1,21 @@
+[flake8]
+# References:
+# https://flake8.readthedocs.io/en/latest/user/configuration.html
+# https://flake8.readthedocs.io/en/latest/user/error-codes.html
+# https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
+exclude = __init__.py,versioneer.py
+ignore =
+    E203,  # whitespace before ':'
+    E231,E241,  # Multiple spaces around ","
+    E731,  # Assigning lambda expression
+    #E741,  # Ambiguous variable names
+    W503,  # line break before binary operator
+    W504,  # line break after binary operator
+    F821,  # undefined name
```

Comment:
This exclusion is to resolve failures that come when we try to specify a param type with a class from a module we haven't imported; for example:

https://github.com/dask-contrib/dask-sql/blob/4d5f7dd4c133f7f2c584db2492c8f0ef3cba7e31/dask_sql/physical/rel/convert.py#L35-L37

Are we okay with keeping this exclusion, or would we prefer importing these modules where this failure comes up?